apps/demo-app/src/styles/modules/theming/_theming.scss

```diff
@@ -3,22 +3,22 @@
 
 $theme: (
 	header: (
-		header: (
-			background: vars.$primary-lightest,
-			title-size: 32px,
-		),
-		button-primary: (
-			default-background: vars.$primary,
-			default-font-color: vars.$font-color-light,
-			hover-background: vars.$accent,
-			hover-font-color: vars.$font-color-light,
-		),
-		button-accent: (
-			default-background: vars.$accent,
-			default-font-color: vars.$font-color-light,
-			hover-background: vars.$primary,
-			hover-font-color: vars.$font-color-light,
-		)
+		// // header: (
```

Comment:
fell free to remove theis commented in header theme config. It will not be in use anymore, right?